modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/DefaultParameterExtension.java

```diff
@@ -237,4 +239,12 @@ protected boolean shouldIgnoreClass(Class<?> cls) {
         return cls.getName().startsWith("javax.ws.rs.");
     }
 
+    private static Iterable<Annotation> annotations(AnnotationMap annotationMap) {
+        if (annotationMap == null) {
+            return Collections.emptyList();
+        } else {
+            return annotationMap.annotations();
+        }
+    }
+
```

Comment:
see also general review, we would keep as is to maintain support for jackson 2.8.x